presto-docs/src/main/sphinx/security/hive.rst

```diff
@@ -0,0 +1,237 @@
+===========================
+Hive Security Configuration
+===========================
+
+When set up without specific configuration options, the :doc:`/connector/hive`
```

Comment:
I think it reads best when talking about default behavior. "The default security mode of the hive connector is to use simple authentication."